config/locales/de.yml

```diff
@@ -398,6 +414,9 @@ de:
           has_one: Datensatz kann nicht gelöscht werden, da ein abhängiger %{record}-Datensatz
             existiert.
           has_many: Datensatz kann nicht gelöscht werden, da abhängige %{record} existieren.
+    attributes:
```

Comment:
Where does this get used ? (Asking for my information)

Reply:
I think it's a default translation when a model attribute gets translated somewhere in a view file. Rails comes with this magic lookup path for attributes. It's good to investigate though where this is actually being used.

Reply:
It’s used e.g. for validation error messages. For example, if a contributor enters an invalid phone number.